worlds/AutoWorld.py

```diff
@@ -132,6 +136,10 @@ class World(metaclass=AutoWorldRegister):
     item_names: Set[str]  # set of all potential item names
     location_names: Set[str]  # set of all potential location names
 
+    # If the game displays all contained items to the user, this flag pre-fills the hint system with this information
```

Comment:
this is gone. PR should not re-add it.

Reply:
Please be more specific with your feedback. What is "this"?

Reply:
The 4 lines this PR is adding at this marker point. sending_visible is no longer a thing.

Reply:
Those lines have been removed. I'm not actually sure how they snuck back in there, considering I never touched them while working on the WebHost.